src/main/resources/application.yaml

```diff
@@ -10,4 +13,6 @@ spring:
         driver-class-name: com.mysql.cj.jdbc.Driver
     flyway.baseline-on-migrate: true
     jpa:
-        hibernate.ddl-auto: validate
\ No newline at end of file
+        hibernate.ddl-auto: validate #update
+    main:
+        allow-circular-references: true
```

Comment:
Normally you do not want this enabled. This option generally means that you have some serious architecture issues when two modules depend on each other
Remove it and try to fix whatever the reason was to have it enabled